README.md

```diff
@@ -124,10 +125,7 @@ RubyCritic is supporting:
 * 2.0
 * 2.1
 * 2.2
-
-Note that we do not support Ruby 1.9 anymore.
```

Comment:
With Ruby's 2.0 having reached the end of its lifetime I believe we seriously need to encourage people to use an up to date Ruby. I think we shouldn't litter our code and doc base with 1.9 references anymore.